Intersect.Client.Framework/Entities/IAnimation.cs

```diff
@@ -3,24 +3,17 @@
 
 namespace Intersect.Client.Framework.Entities
 {
-    public interface IAnimation
+    public interface IAnimation : IDisposable
     {
         bool AutoRotate { get; set; }
         bool Hidden { get; set; }
         bool InfiniteLoop { get; set; }
         AnimationBase MyBase { get; set; }
+        Point Size { get; }
 
-        Point AnimationSize();
-        void Dispose();
-        bool Disposed();
-        void DisposeNextDraw();
-        void Draw(bool upper = false, bool alternate = false);
-        void EndDraw();
         void Hide();
-        bool ParentGone();
```

Comment:
Was this not useful (legitimate question, I have no idea)?

Reply:
Lord knows, never used it.
All it does is check if Parent is disposed or null.